store/account.ts

```diff
@@ -189,10 +189,13 @@ export const actions = actionTree(
       commit('setSubaccountBalancesWithToken', subaccountBalancesWithToken)
     },
 
-    async fetchSubaccountsBalancesWithPrices({ commit, state }) {
+    async fetchSubaccountsBalancesWithPrices(
+      { commit, state },
+      { refresh }: { refresh: boolean } = { refresh: false }
```

Comment:
Can you please refactor this code in some other way, passing a boolean to ensure something is run is not ideal - we shouldn't have to worry about whether the balances are in the state.